app/assets/src/components/common/MetadataCSVLocationsMenu.jsx

```diff
@@ -156,7 +156,9 @@ class MetadataCSVLocationsMenu extends React.Component {
             isHuman={isRowHuman(row)}
             warning={CSVLocationWarnings[sampleName]}
           />
-          {applyToAllSample === sampleName && this.renderApplyToAll()}
+          {applyToAllSample === sampleName &&
+            this.props.metadata.rows > 1 &&
```

Comment:
this should be `this.props.metadata.rows.length`, right?

Reply:
good catch! I hadn't tested that yet 😛